website/src/components/views/compareSideBySide/GenericCompareSideBySideReactPage.tsx

```diff
@@ -0,0 +1,87 @@
+import { type FC, useMemo } from 'react';
+
+import { GenericCompareSideBySideDataDisplay } from './GenericCompareSideBySideDataDisplay.tsx';
+import { toDownloadLink } from './toDownloadLink';
+import { type OrganismsConfig } from '../../../config';
+import { OrganismViewPageLayout } from '../../../layouts/OrganismPage/OrganismViewPageLayout.tsx';
+import { type OrganismViewKey, type OrganismWithViewKey, Routing } from '../../../views/routing';
+import { compareSideBySideViewKey } from '../../../views/viewKeys';
+import { CompareSideBySidePageStateSelector } from '../../pageStateSelectors/CompareSideBySidePageStateSelector';
+
+export type GenericCompareSideBySideReactPageProps = {
+    organism: OrganismWithViewKey<typeof compareSideBySideViewKey>;
+    hideMutationComponents?: boolean;
+    organismsConfig: OrganismsConfig;
+    isStaging: boolean;
+};
+
+export const GenericCompareSideBySideReactPage: FC<GenericCompareSideBySideReactPageProps> = ({
+    organism,
+    hideMutationComponents,
+    organismsConfig,
+    isStaging,
+}) => {
+    const organismViewKey = `${organism}.${compareSideBySideViewKey}` satisfies OrganismViewKey;
+
+    const view = useMemo(
+        () => new Routing(organismsConfig).getOrganismView(organismViewKey),
+        [organismsConfig, organismViewKey],
+    );
```

Comment:
The same memo is also in the state selector and in the data display, but I wanted to keep it simple for now. The text step will be to move all this to a shared state and get rid of the duplication.